definitions/variable/land/land-cover.yaml

```diff
@@ -0,0 +1,57 @@
+- Land Cover:
+    definition: Total land cover
+    unit: million ha
+    tier: 2
+    agmip: Land Cover (LAND) - Total (TOT)
+- Land Cover|Built-Up Area:
+    definition: Built-up land associated with human settlements
+    unit: million ha
+    tier: 2
+    engage: Land Cover|Built-up Area
+    navigate: Land Cover|Built-up Area
+- Land Cover|Pasture:
+    definition: Pasture, not only high quality rangeland, based on FAO definition of
+      "permanent meadows and pastures"
+    unit: million ha
+    tier: 2
+- Land Cover|Cropland:
+    definition: Arable land, i.e. land in bioenergy crop, food, and feed/fodder crops,
+      permanent crops as well as other arable land (physical area)
+    unit: million ha
+    tier: 2
+    agmip: Land Cover (LAND) - All crops (CRP)
+- Land Cover|Forest:
+    definition: Managed and unmanaged forest area
+    unit: million ha
+    tier: 2
+    agmip: Land Cover (LAND) - Forest (FOR)
+
+- Land Cover|Water Ecosystems:
```

Comment:
I suggest to move Water Ecosystems to seperate file and remove the prefix "Land Cover" like `Water Ecosystems|Forests`
We need peatlands as a subtype of wetlands
Water Ecosystems|Wetlands|Peatlands
Water Ecosystems|Wetlands|Peatlands|Intact
Water Ecosystems|Wetlands|Peatlands|Drained
Water Ecosystems|Wetlands|Peatlands|Rewetted

Reply:
Removed